presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -678,6 +678,10 @@ protected RowExpression visitInPredicate(InPredicate node, Void context)
         {
             ImmutableList.Builder<RowExpression> arguments = ImmutableList.builder();
             arguments.add(process(node.getValue(), context));
+            if (node.getValueList() instanceof SymbolReference) {
```

Comment:
I thought about this as well. I'm just not sure originally where `SymbolReference` got replaced by `InList`. Could you add a comment here to explain?

Reply:
The related test is testDoubleNestedCorrelatedSubqueries. The query (nested correlated subqueries) in the test
```
SELECT orderkey FROM orders o WHERE 3 IN (SELECT o.custkey FROM lineitem l WHERE (SELECT l.orderkey = o.orderkey))
```
 is not supported today (tested in master branch).  Let me try to remove this from translator and do not do a expression to row expression translation for this test. We need to think how to fix this test in longer term though.